src/providers/twitch/twitchapi.hpp

```diff
@@ -0,0 +1,17 @@
+#pragma once
+
+#include "common/Singleton.hpp"
+
+#include <QString>
+
+namespace chatterino {
+
+class TwitchApi
+{
+public:
+    static void FindUserId(const QString user, std::function<void(QString)> callback);
```

Comment:
Follow the normal function and method naming convention, which would in this case be `findUserId`